### ISSUE TITLE
Option to pause on incoming call

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#47]
```
There should be an option to pause the music if there's an incoming call.
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2011-09-24 09:25:46

**Comment by kaaholst** [comment]
```
One possible implementation?
http://stackoverflow.com/questions/5610464/stopping-starting-music-on-incoming-calls
```

Original issue reported on code.google.com by `drdocmatt` on 2014-03-17 10:27:39

**Comment by kaaholst** [comment]
```
Yeah, the implementation's reasonably straightforward.

The reason I haven't done this (despite filing the feature request in the first place)
is that to do this you need the READ_PHONE_STATE permission in Android, which pops
up a somewhat scary (and for good reason) warning to people before they install the
app.

I figure that the tiny benefit of this feature isn't worth putting people off installing
Squeezer ("Wait, why does this application that's a remote control need access to my
phone details?").

Especially when there are other things that might cause you want to pause the music
(e.g., if your landline phone rings) that this doesn't solve.  So I think the effort
would be better spent improving Squeezer's notifications (so you can pause/play from
the notification message) and have a lockscreen widget (perhaps via integration with
Dashclock).
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2014-03-17 11:02:33

**Comment by kaaholst** [comment]
Totally worthwhile feature, go for it!
Yes there are other possible interruptions, but this is a common one, and once your phone is ringing it can take a lot of rushed swiping to get to the app, silence the music and get back to to the answer screen before it goes to voicemail.

Original issue reported on github.com/nikclayton/android-squeezer by blscorp on 2020-10-06T16:43:43Z

**Comment by kaaholst** [comment]
This will be available in the next version of Squeezer which is in beta now. 
You can opt in to beta [here](https://play.google.com/apps/testing/uk.org.ngo.squeezer). 
You can opt out of beta any time.

Original issue reported on github.com/nikclayton/android-squeezer by kaaholst on 2021-01-11T19:09:41Z

**Comment by kaaholst** [comment]
That's great!  Nice going, I'll wait for the update.
Thanks for the alert.

Original issue reported on github.com/nikclayton/android-squeezer by blscorp on 2021-01-11T19:55:31Z

**Comment by kaaholst** [comment]
This is available in Squeezer version 2.2.2, so closing this issue.

Original issue reported on github.com/nikclayton/android-squeezer by kaaholst on 2021-01-23T20:02:11Z
